src/com/epam/javacourse/homework4/GreenhouseManagement.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.javacourse.homework4;
+
+public class GreenhouseManagement implements Greenhouse {
+
+    public GreenhouseManagement() {
+    }
+
+    @Override
+    public void buyPlant(Plant plant) {
+            System.out.println("Plant is bought.");
+    }
+
+    @Override
+    public void waterPlants() {
```

Comment:
This method does not affect the `Plant` instance. Maybe it could change the `isWatered` parameter to true or do something else?